tests/integration/states/file.py

```diff
@@ -579,6 +579,27 @@ def test_managed_contents(self):
             for typ in managed_files:
                 os.remove(managed_files[typ])
 
+    @destructiveTest
+    def test_managed_check_cmd(self):
+        '''
+        Test file.managed passing a basic check_cmd kwarg. See Issue #38111.
+        '''
+        ret = self.run_state(
+            'file.managed',
+            name='/tmp/sudoers',
```

Comment:
Thanks @cachedout - Good catch. The `mode` kwarg doesn't work on Windows anyway, so I have put a `skipIf` on this test saying so.